testsuite/oauth/refresh_token.py

```diff
@@ -46,13 +41,12 @@ def revoke_access_token(self):
         post_data = {
             'token': self.access_token,
         }
-        response = requests.delete(self._config['token_url'],
+        response = requests.delete(self._urls['token'],
```

Comment:
Are you still planning to nix the `revoke_access_token` method here? (**Eventually** we'll want one but it will pretty much have to be webdriver-controlled to deal with UX variability in the portals.)